samples/drivers/display/boards/mimxrt1050_evk.conf

```diff
@@ -0,0 +1 @@
+CONFIG_DISPLAY_MCUX_ELCDIF=y
```

Comment:
This shouldn't be necessary because of:
https://github.com/zephyrproject-rtos/zephyr/blob/d58f8bee79dddb9aa1b65a2ef1624ebf1b389a4e/soc/arm/nxp_imx/rt/Kconfig.defconfig.series#L28-L33

Reply:
removed